src/entity/models/entityManager.ts

```diff
@@ -34,6 +34,26 @@ export class EntityManager {
     await this.repository.insert(newEntity);
   }
 
+  public async createEntities(newEntities: IEntity[]): Promise<void> {
+    this.logger.log('info', `creating bulk entities`);
+
+    const dbEntity = await this.repository.findOne({
+      where: [
+        {
+          externalId: In(newEntities.map((entity) => entity.externalId)),
+        },
+        { osmId: In(newEntities.map((entity) => entity.osmId)) },
+      ],
+    });
+
+    if (dbEntity) {
+      const message = `an entity with the following ids: ${JSON.stringify(dbEntity)} already exists`;
+      throw new IdAlreadyExistsError(message);
+    }
+
+    await this.repository.insert(newEntities);
+  }
+
```

Comment:
does it insert in a single transaction or X transactions?

Reply:
it inserts in a single transaction. here's an example from the query logger:
```
query: INSERT INTO "entity"("external_id", "osm_id") VALUES ($1, $2), ($3, $4) -- PARAMETERS: ["3fa8534-5717-4562-b3fc-2c963f26a1a6",1233,"3fa85f3-5717-4562-223fc-2c963f66afa6",1430]
```